modules/preview-react/text-area/stories/examples/LabelPositionVertical.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {TextArea} from '@workday/canvas-kit-preview-react/text-area';
+
+export const LabelPositionVertical = () => {
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setValue(event.target.value);
+  };
+
+  return (
+    <TextArea orientation='vertical' spacing='m'>
```

Comment:
```suggestion
    <TextArea orientation='vertical'>
```

I think this is a mistake if I'm not mistaken? haha

Reply:
In the mdx doc I have:

> Use the `orientation` property to set `TextInput.Label`'s position. You can override the default
spacing using the `spacing` prop. Below are examples of both positions:

Thought it might be worth adding an example of spacing override in vertical story since basically all the other stories already show vertical orientation.

Reply:
Ah - sorry I missed that because the horizontal example was between the vertical example and that note. Can we at least manually pass in the default spacing (`xxxs`) so that the visual doesn't look extra spacey?

![image](https://user-images.githubusercontent.com/908478/142670958-25190cf9-0b17-4e0e-994e-aec782caaf68.png)

Reply:
@anicholls Done!